configs/lad/metafile.yml

```diff
@@ -6,7 +6,7 @@ Collections:
         - Label Assignment Distillation
         - SGD with Momentum
         - Weight Decay
-      Training Resources: 8x V100 GPUs
+      Training Resources: 2x V100 GPUs
```

Comment:
if use 2 GPUs, we may need to change the config name

Reply:
done